src/js/components/Form/stories/AllChildren.js

```diff
@@ -40,14 +40,21 @@ const Example = () => (
               ]}
             />
           </FormField>
-          <FormField name="subscribe">
-            <CheckBox name="subscribe" label="Subscribe?" />
+          <FormField name="subscription">
+            <CheckBoxGroup
+              name="subscription"
+              options={['subscribe', 'receive email notifications']}
+            />
           </FormField>
           <FormField name="ampm">
             <RadioButtonGroup name="ampm" options={['morning', 'evening']} />
           </FormField>
           <FormField label="Size" name="size">
-            <Select name="size" options={['small', 'medium', 'large']} />
+            <Select
+              name="size"
+              multiple
```

Comment:
This was just for the sake of comparing the data structures of Select multiple with CheckBoxGroup, I'm ok removing `multiple` from this example.